spec/integ/matrix-client-crypto.spec.ts

```diff
@@ -240,7 +234,7 @@ function bobSendsReplyMessage() {
  * @return {promise} which resolves to the ciphertext for Bob's device.
  */
 function expectAliSendMessageRequest() {
-    return expectSendMessageRequest(aliTestClient.httpBackend).then(function(content) {
+    return expectSendMessageRequest(aliTestClient.httpBackend).then(function(content: IContent) {
```

Comment:
all these inline functions should probably be converted to arrow functions, but not the end of the world.